docs/src/development/headers.md

```diff
@@ -10,22 +10,23 @@ sidebarTitle: "Headers"
 
 ## Request headers
 
-Platform.sh adds the following HTTP headers in the router to give the application information about the connection.  These are stable and may be examined by the application as necessary.
+Platform.sh adds the following HTTP headers in the router to give the application information about the connection. These are stable and may be examined by the application as necessary.
 
 * `X-Forwarded-Proto`: The protocol forwarded to the application, e.g. "http", "https".
```

Comment:
To avoid the linting errors, just enclose this values in code markers.
```suggestion
* `X-Forwarded-Proto`: The protocol forwarded to the application, for example `http`, `https`.
```